e2e/pnpm_workspace/BUILD.bazel

```diff
@@ -0,0 +1,3 @@
+load("@npm//:link_js_packages.bzl", "link_js_packages")
```

Comment:
have you already bike-shedded this load path? should there be a `defs.bzl` inside the generated external/npm/BUILD.bazel file? this is the part users will type/paste a lot right?

Reply:
agreed; `defs.bzl` is best for this one as it will be most common load point on the happy path